main.py

```diff
@@ -742,22 +742,29 @@ def make_move(move, board:Board):
     #update king position
     if board.board[position] == K or board.board[position] == k:
         board.king_position[board.side] = target
-    else:
+    #update other pieces positions
+    elif board.board[position] != K or board.board[position] != k:
```

Comment:
why would you check for every piece not K or k instead of just looking for . positions with the if statement?

Reply:
first I had there else. I just want to update positions of pieces. we talked about it. that it should be elif, not else. I came up with this unnecessary condition

Reply:
Hmm okay i think it would still be better making a seperate if statement for those cases but its doable like that too.